src/main/java/com/epam/training/toto/service/TotoService.java

```diff
@@ -1,4 +1,142 @@
 package com.epam.training.toto.service;
 
+import com.epam.training.toto.domain.Outcome;
+import com.epam.training.toto.domain.Round;
+import com.epam.training.toto.service.parser.CSVParser;
+import com.epam.training.toto.service.parser.util.OutcomeParser;
+
+import java.io.IOException;
+import java.text.NumberFormat;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.*;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
 public class TotoService {
+    public static final Integer NUMBER_OF_GAMES = 14;
+    private final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy.MM.dd.");
+    private final NumberFormat numberFormat = NumberFormat.getCurrencyInstance(new Locale("uk", "UA"));
+
+    private Map<LocalDate, Round> data;
+
+    public TotoService() {
+        try {
+            data = new CSVParser().readData();
```

Comment:
I'd not suggested use hard-weight execution in constructor. Also it's bad practice to throw RuntimeException from constructor. Use separate method for that.

Reply:
add custom DataCanNotBeReadException
take parser as a private field
don't implement lazy initialization of data because in that case we would read .csv 3 times without any reason
extract initData() to App